auth-web/src/components/auth/manage-business/EntityDetailsAlert.vue

```diff
@@ -0,0 +1,112 @@
+<!-- TODO:
```

Comment:
Can you use the IconTooltip component in here? You will need to add some more props to it, but that way we only have the one template for icon tooltips. You could add a slot to the content as well to make it easy to add your list in there

Reply:
I think I would also need to change some of the CSS too since the common tooltip comes out of the bottom and points upward while we want this tooltip to come out of the top and point downward. 

can I do that through props and stuff?

Reply:
![image](https://user-images.githubusercontent.com/82549206/215818143-63bc44ce-0f12-4d0f-a02a-e6fe7a4c12f3.png)

vs

![image](https://user-images.githubusercontent.com/82549206/215818196-17efe360-e4bc-4112-b655-3e932659e0e1.png)

Reply:
yep just add the css you need for the top variant and set the class/html variant via a prop. It will be really good to have that option for future things

Reply:
yeah made 2 classes to toggle between top and bottom tooltips